recipes/weblogo/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "weblogo" %}
+{% set version = "3.7.8" %}
+{% set license = "MIT" %}
+{% set license_file = "LICENSE.txt" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/weblogo-{{ version }}.tar.gz
+  sha256: c6f8ff118ec0c8401a7d4c3c111c0108eee8523b092afe0dccec42e8134622f3
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - weblogo = weblogo._cli:main
+    - transformseq = weblogo._transformseq:main
+  script_env:
+    ## NOTE:
+    # https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#inherited-environment-variables
+    - SEP_CHAR=/  # [not win]
+    - SEP_CHAR=\\  # [win]
```

Comment:
I don't think this is needed. `conda-build` should recognize forward slashes in windows too.

Reply:
And even if it is, you could just define a jinja var instead of an env var.

Reply:
But the build scripts for windows use `\` on the maintainer docs. Please see my comment here: [(comment)](https://github.com/conda-forge/staged-recipes/pull/17283#discussion_r783412211).